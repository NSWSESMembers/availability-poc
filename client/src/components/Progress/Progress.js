import React from 'react';
import { ActivityIndicator, View } from 'react-native';

import styles from './styles';

const Progress = () => (
  <View style={styles.container}>
    <ActivityIndicator size="large" />
  </View>
);

export default Progress;
