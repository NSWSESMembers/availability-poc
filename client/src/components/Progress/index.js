import Progress from './Progress';
import styles from './styles';

export { Progress, styles };
