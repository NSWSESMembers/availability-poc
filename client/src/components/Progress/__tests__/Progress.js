import React from 'react';
import renderer from 'react-test-renderer';

import Progress from '../Progress';

test('renders correctly', () => {
  const tree1 = renderer.create(
    <Progress />,
  ).toJSON();
  expect(tree1).toMatchSnapshot();
});
