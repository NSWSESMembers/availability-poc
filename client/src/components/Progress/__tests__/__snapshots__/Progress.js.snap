// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
